### PR TITLE
New line in druhy_bok

### PR DESCRIPTION
**Pull request by bretislavstepan**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2716)

### CODE REVIEW

File: czech/druhy_bok/spanek.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 Spíš, ale ve spánku si uvědomuješ sama sebe. Divné. Dokážeš ovlivnit své myšlenky. Co uděláš?
 Spoustu lidí chce létat a ve spánku, pokud jsme si vědomi sami sebe, můžeme. Existuje 
-i odborný termín - anglicky lucid dreaming.
\ No newline at end of file
+i odborný termín - anglicky lucid dreaming.
+
+[Pokusíš se usnout ve snu](sen/senvesnu.md)
```
Comment (casey-collab) [inline czech/druhy_bok/spanek.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: czech/druhy_bok/sen/ve_snu.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Ve spánku jsis uvědomil sám se a pokusil jsi se usnout. To se ti povedlo. Zdá se ti sen.
```
Comment (casey-collab) [inline czech/druhy_bok/sen/ve_snu.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by bretislavstepan** [comment]
Hi @casey-collab,
I have fixed link and some typo.

**Comment by casey-collab** [comment]
Looks great! :lollipop:

**Pull request merged**